### DISCUSSION TITLE
Supported regions

### DISCUSSION THREAD

**Discussion by landro** [artilleryio/artillery#2640] [Artillery]
Hi there, 

I'm trying to use my ECS cluster in the eu-north-1 region, but it turns out that region is not supported by artillery. How come?
Does it even make sense to limit the regions where one can run tests from?

Cheers,

Stefan

**Comment by hassy** [comment]
@landro will take a look. The reason the list of regions is explicit is that not all regions have services that we rely on (e.g. SQS FIFO queues).

**Reply by bernardobridge** [reply]
@landro , we'll be releasing a new version today and we'll include `eu-north-1` in the supported regions so it works for you. 

We'll review in the near future the need for this list, as it looks like the features we need are now more widely available.

**Reply by hassy** [reply]
@landro out now in v2.0.10!

**Reply by landro** [reply]
Thanks, @hassy !
